fastn-core/src/auth/email_password.rs

```diff
@@ -540,13 +542,17 @@ async fn create_and_send_confirmation_email(
         .first(&mut conn)
         .await?;
 
+    let file_path = fastn_ds::Path::new("email/confirmation-mail.html");
```

Comment:
This has to be part of the auth package. So read it by ftd. We will store the content of the HTML in a ftd file. `auth.confirmation_mail_html` or something would be our "interface".

Reply:
done https://github.com/fastn-stack/fastn/pull/1741/commits/f4bbbcef39c4d70e09f6b1e22799bd2f3c0fc7b6